# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../openal/hotfix-nommx-support.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- openal/linux/src/mixaudio16.c.~1.4.~	2004-09-10 13:11:40.000000000 +0200
+++ openal/linux/src/mixaudio16.c	2004-10-21 22:29:39.588018832 +0200
@@ -19,7 +19,7 @@
 #define MIN(a,b) ((a) < (b) ? (a) : (b))
 #define MAX(a,b) ((a) > (b) ? (a) : (b))
 
-
+#undef MMX_SUPPORT
 #ifdef MMX_SUPPORT
 #error MMX support needs more work.  Do not enable it unless you plan to fix it.
 #endif
